orchestrator/ethereum_gravity/src/send_erc721_to_cosmos.rs

```diff
@@ -0,0 +1,115 @@
+//! Helper functions for sending tokens to Cosmos
+
+use clarity::abi::{encode_call, Token};
+use clarity::PrivateKey as EthPrivateKey;
+use clarity::{Address, Uint256};
+use deep_space::address::Address as CosmosAddress;
+use gravity_utils::error::GravityError;
+use std::time::{Duration, Instant};
+use web30::client::Web3;
+use web30::types::SendTxOption;
+
+pub const SEND_TO_COSMOS_GAS_LIMIT: u128 = 100_000;
+ 
+#[allow(clippy::too_many_arguments)]
+pub async fn send_erc721_to_cosmos(
+    erc20: Address,
+    gravity_contract: Address,
+    amount: Uint256,
+    cosmos_destination: CosmosAddress,
+    sender_secret: EthPrivateKey,
+    wait_timeout: Option<Duration>,
+    web3: &Web3,
+    options: Vec<SendTxOption>,
+) -> Result<Uint256, GravityError> {
+    print!("Now in the send_erc721_to_cosmos function!");
+    let sender_address = sender_secret.to_address();
+    let mut approve_nonce = None;
```

Comment:
This function should start with a query for this stat variable https://github.com/Gravity-Bridge/Gravity-Bridge/blob/main/solidity/contracts/GravityERC721.sol#L14

see the similar getters in the utils file of ethereum_gravity. This will produce an error if the contract is not GravityERC721.sol, I say this becuase mixups with Gravity.sol's address are highly probable.

Reply:
Added utility in ethereum_gravity: pub async fn get_gravity_sol_address()
Then am checking the gravity sol address in gravityERC721.sol like so: 

```
    let grav_sol_address_in_erc721 = get_gravity_sol_address(gravity_erc721_address, *MINER_ADDRESS, web30)
    .await
    .unwrap();
    
    assert_eq!(grav_sol_address_in_erc721, gravity_address);
```